src/app/core/services/intel.service.ts

```diff
@@ -0,0 +1,240 @@
+ * @param n The intel to map.
+ */
+function appIntelFromNet(n: NetIntel): Intel {
+  switch (n.type) {
+    case 'analog-radio-message':
+      return appAnalogRadioMessageIntelFromNet(n);
+    case 'plaintext-message':
+      return appPlaintextMessageIntelFromNet(n);
+    default:
+      throw new MDSError(MDSErrorCode.AppError, `unsupported intel type while converting to app representation: ${ (n as Intel).type }`);
+  }
+}
+
+@Injectable({
+  providedIn: 'root',
+})
+
+/**
+ * Service for intel management. Allows creation, manipulation and retrieval of intel.
+ */
+export class IntelService {
+
+  constructor(private netService: NetService) {
+  }
+
+  /**
+   * Creates new intel and returns the created one.
+   * @param create The intel to create.
+   */
+  createIntel(create: CreateIntel): Observable<Intel> {
+    interface NetCreate {
+      operation: string;
+      type: string;
+      content: object;
+      importance: number;
+      initial_deliver_to: string[];
+    }
+
+    const body: NetCreate = {
+      operation: create.operation,
+      type: create.type,
+      content: create.content,
+      importance: create.importance,
+      initial_deliver_to: create.initialDeliverTo,
+    };
+
+    return this.netService.postJSON<NetIntel>('/intel', body, {}).pipe(
+      map((res: NetIntel): Intel => appIntelFromNet(res)),
+    );
+  }
+
+  /**
+   * Invalidates the given intel.
+   * @param intelId The id of the intel to invalidate.
+   */
+  invalidateIntel(intelId: string): Observable<void> {
+    return this.netService.postJSON(urlJoin('/intel', intelId, 'invalidate'), {}, {});
+  }
+
+  /**
+   * Retrieves the intel with the given id.
+   * @param intelId The id of the intel to be retrieved.
+   */
+  getIntelById(intelId: string): Observable<Intel> {
+    return this.netService.get<NetIntel>(urlJoin('/intel', intelId), {}).pipe(
+      map((res: NetIntel): Intel => appIntelFromNet(res)),
+    );
+  }
+
+  /**
+   * Retrieves paginated {@link Intel} list.
+   * @param params Params for pagination.
+   * @param filters Filters for retrieval.
+   */
+  getIntel(params: PaginationParams<any>, filters: IntelFilter): Observable<Paginated<Intel>> {
+    interface NetParams extends NetPaginationParams {
+      created_by?: string,
+      operation?: string,
+      intel_type?: string,
+      min_importance?: number,
+      include_invalid?: boolean,
+      one_of_delivery_for_entries?: string[],
+      one_of_delivered_to_entries?: string[],
+    }
+
+    const nParams: NetParams = netPaginationParams(params, () => undefined);
+    nParams.created_by = filters.createdBy;
+    nParams.operation = filters.operation;
+    nParams.intel_type = filters.intelType;
+    nParams.min_importance = filters.minImportance;
+    nParams.include_invalid = filters.includeInvalid;
+    nParams.one_of_delivery_for_entries = filters.oneOfDeliveryForEntries;
+    nParams.one_of_delivered_to_entries = filters.oneOfDeliveredToEntries;
```

Comment:
```suggestion
    const nParams: NetParams = {
      ...netPaginationParams(params, () => undefined),
      created_by: filters.createdBy,
      operation: filters.operation,
      ...
  }
```
Nicht unbedingt nötig, aber vielleicht schicker ;)